packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java

```diff
@@ -517,6 +520,38 @@ public void onResponse(Call call, Response response) throws IOException {
     }
   }
 
+  @ExpoMethod
+  public void getContentUriAsync(String uri, Promise promise){
+    try{
+      final Uri fileUri = Uri.parse(uri);
+      ensurePermission(fileUri, Permission.WRITE);
+      ensurePermission(fileUri, Permission.READ);
+      checkIfFileDirExists(fileUri);
+      if("file".equals(fileUri.getScheme())){
+        File file = uriToFile(fileUri);
+        Bundle result = new Bundle();
+        result.putString("uri", cUriFromFile(file).toString());
+        promise.resolve(result);
+      }
+      else{
+        promise.reject("E_DIRECTORY_NOT_READ","No readable files with the uri: " + uri);
```

Comment:
If we get here, `fileUri.getScheme() != 'file'`, maybe the rejection message should prompt the developer to use other URI?